devsetup/scripts/gen-edpm-compute-node.sh

```diff
@@ -181,6 +181,8 @@ sed -i s/dhcp/none/g $NETSCRIPT
 sed -i /PERSISTENT_DHCLIENT/d $NETSCRIPT
 EOF
 
+chmod +x ${OUTPUT_BASEDIR}/${EDPM_COMPUTE_NAME}-firstboot.sh
```

Comment:
I'm not 100% sure how is this related to the ssh-keyscan removal. Maybe it would be a good idea to make a note of it in PR or even in the commit msg.

Reply:
Updated the commit message.